### ISSUE TITLE
library.properties file

### ISSUE THREAD

**Issue by ZanzyTHEbar** [RefreshMyMind-I/Arduion_library_creation_guide#1]
# Great Start

This repo is a great start, i would add a `library.properties` file to the repo to complete the tutorial. 

Secondly, but not required, i would smarten up the `md` it's pretty hard to follow as it is now. :) 

Keep it up!

**Comment by RefreshMyMind-I** [comment]
Thanks for the feedback man, I will look into it.

**Comment by MandoRick** [comment]
Arduion_library_creation_guide, you spelled Arduino wrong.  Love your work bro.  Thank you for making this example.
